release.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+# This script creates a Sia-UI release for all 3 platforms: osx (darwin),
+# linux, and windows.  it takes 3 arguments, each one a version.  The first
+# argument is the electron version to package, the second argument is the Sia
+# version to package, and the third argument is version used to label the
+# Sia-UI release.  The archives are written out to release/.
```

Comment:
the order should probably be reversed, otherwise you need to specify the electron/sia versions whenever you want to specify the ui version, which I imagine would be far more common.

also, use `set -e` so that any error terminates the script.